manifests/windows.pp

```diff
@@ -0,0 +1,55 @@
+# Local User - Windows Edition
+#
+# DESCRIPTION:
+# WIP - Allows you to set local users and RDP access via Puppet (for now...)
+#  
+# USAGE: 
+# local_user::windows { 'steve' :
+#   state     => present|absent, # currently set to present or absent
+#   password  => <your password>, # a plain text password
+#   comment   => "Full Name", # user's full name
+#   groups    => <string or array of groups>, # add user to groups
+#   allow_rdp => true|false,  # grant user rdp access
+#   admin     => true|false,  # add user to Administrators group
+# }
+#
+# local_user::windows { 'bob' :
+#   state     => present,
+#   password  => 'Bobbo1234',
+#   groups    => ['Administrators'],
+#   comment   => 'Bob is Cool',
+#   allow_rdp => true,
+# }
+
+define local_user::windows (
+  $state,
+  $password,
+  $groups = [],
+  $admin = false,
+  $comment = $name,
+  $allow_rdp = true,
+) {
+
+  if $admin == true and $allow_rdp == true {
```

Comment:
I might wrap this into one `if` statement. 

```puppet
if ($admin) {
    if ($allow_rdp) {
    }
    else {
    }
else { 
    if ($allow_rdp) {
    }
}
```

@rnelson0 might have a differing opinion, though.